script/setup-kind.sh

```diff
@@ -16,6 +16,11 @@ install_metrics() {
   # https://github.com/kubernetes-sigs/kind/issues/398#issuecomment-621143252
   kubectl apply -f "$(dirname "$0")/kind-metrics-server.yaml"
 
+  kubectl wait pod --namespace kube-system \
```

Comment:
`wait` is neat!

Reply:
Totally!  I didn't know about it - I saw it in the `kind` docs.